fecgov/openFEC
# drop unused database objects; alter ownership

## Summary (required)

- Resolves #[_3866_](https://github.com/fecgov/openFEC/issues/3866)

- This PR will drop four database objects that are not being used anymore, and removed them from refresh list too.   A statement on ownership/privilege that was missed in a earlier ticket is added here.

## How to test the changes locally

- Download the feature branch
- run pytest to make sure all tests pass
- run flyway migrate on local, then log into local cfdm_test,  these four objects should not exist in public schema
public.ofec_report_pac_party_all_vw;
public.ofec_reports_pacs_parties_vw;
public.ofec_report_pac_party_all_mv;
public.ofec_reports_pacs_parties_mv

owner of pubic.ofec_reports_pac_party_vw shoud be fec.
- test materialized views refreshing job, make sure SQLA_SAMPLE_DB_CONN point to local cfdm_test:
      dropdb cfdm_test
      createdb cfdm_test
      invoke create_sample_db
- these two mvs should not be on the refreshing list:
   ofec_report_pac_party_all_mv
   ofec_reports_pacs_parties_mv
